redhat-cop/casl-ansible
# Allow for non-privileged requests

#### What does this PR do?
Introducing the use of `oc create` to allow for non-privileged requests through the use of objects such as "ProjectRequest". 

#### How should this be manually tested?
Build an inventory that includes "ProjectRequest" objects, but no namespaces or policies. Verify that the deployment can be completed. 

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl @logandonley @etsauer
